### PR TITLE
[DO-1257] Implement Quality alerts page designs

### PR DESCRIPTION
**Pull request by viannbreslerAccso**
Implementation of the quality alerts page figma designs

### CODE REVIEW

File: frontend/src/app/modules/shared/components/table/table.component.scss [hunk 1]
```diff
@@ -192,6 +197,11 @@ table {
   padding-bottom: 8px;
 }
 
+.tableScroll {
+  height: 547px !important;
```
Comment (JoelDietz) [inline frontend/src/app/modules/shared/components/table/table.component.scss:201]: This also applies on the dashboard.
So the height is not allowed to be fixed here.

**Review by JoelDietz — CHANGES_REQUESTED** [review]
The way the height is fixed for the table scrollbar also breaks the dashboard tables. Please change the scrollbar settings.

File: frontend/src/app/modules/shared/components/table/table.component.scss (hunk 1 above)
Comment (viannbreslerAccso) [reply frontend/src/app/modules/shared/components/table/table.component.scss:201]: Fixed

Comment (JoelDietz) [reply frontend/src/app/modules/shared/components/table/table.component.scss:201]: Done

**Review by JoelDietz — APPROVED** [review]

**Pull request merged**